src/App.vue

```diff
@@ -207,7 +207,7 @@ footer {
     min-width: 70%;
 }
 .md-toolbar .md-title .md-input-container:after {
-    background: none ;
+    background: none !important;
```

Comment:
can we make more specified selector, add additional class, instead of using important.